spec/till_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'till'
+require 'timecop'
+
+describe Till do
+
+  let(:till) { described_class.new( { shopName: "The Coffee Connection",
+                                      address: "123 Lakeside Way",
+                                      phone: "16503600708",
+                                      prices: { "Cafe Latte"=> 4.75, "Flat White"=> 4.75 }
+                                     } ) }
+
+  let(:order) { double(:order, customers: ["Jane", "John"], 
+                               items: ["Cafe Latte", "Cafe Latte", "Flat White"],
+                               table: 1 ) }
+
+  let(:amount_discount) { double(:discount, type: :amount, trigger: 10.00 , percent: 10.00) }
+  let(:line_discount) { double(:discount, type: :line, line: "Latte", percent: 5.00) }
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)